src/Endpoints/Indexes.php

```diff
@@ -151,12 +151,12 @@ public function delete(): array
 
     public function getTask($uid): array
     {
-        return $this->http->get(self::PATH.'/'.$this->uid.'/tasks'.'/'.$uid);
+        return $this->http->get('/tasks/'.$uid, ['indexUid' => $this->uid]);
```

Comment:
On the general issue it specified the following point:
> Remove GET `/indexes/:indexUid/tasks/:taskUid`. Use GET `/tasks/:taskUid` instead.

So shouldn't it be something like
```suggestion
        return $this->http->get('/tasks'.'/'.$uid);
```

Reply:
Even better you can use the one from the client maybe

Reply:
@alallema is there any particular reason to suggest:

```php
('/tasks'.'/'.$uid);
```
instead of:

```php
('/tasks/'.$uid);
```